### PR TITLE
kernel, docker, ecs-agent, containerd package updates

### PR DESCRIPTION
**Pull request by etungsten**
**Issue number:**
N/A

**Description of changes:**
```
* 9acc9f39 - Update ecs-agent to 1.57.0 (22 minutes ago) <Sean P. Kelly>
* 6ae84851 - Update containerd to v1.5.8 (41 minutes ago) <Erikson Tung>
* ef96ad2f - Update docker-engine to v20.10.11 (57 minutes ago) <Erikson Tung>
* 0fe7622a - Update kernel-5.4 to 5.4.156 (89 minutes ago) <Matt Briggs>
* b6b1cc21 - Update kernel-5.10 to 5.10.75 (2 hours ago) <Matt Briggs>
```

**Testing done:**
- [x] Built aws-ecs-1 variant, launched ECS instance and ran a nginx task successfully.
- [x] Built aws-k8s-1.19 variant, launched as a worker node and connects to the cluster and runs a pod fine.
- [x] Built aws-k8s-1.21 variant, launched as a worker node and connects to the cluster and runs a pod fine.
- [x] Built vmware-k8s-1.21 variant. launched VM and the node connected to vsphere cluster fine and runs a pod fine.
- [x] Built aws-k8s-1.21 aarch64 image and the node runs pods fine.
- [x] Built aws-ecs-1 aarch64 image, and a task runs successfully.

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

File: packages/docker-engine/docker-engine.spec [hunk 1]
```diff
@@ -3,7 +3,7 @@
 %global goorg github.com/docker
 %global goimport %{goorg}/docker
 
-%global gover 20.10.7
+%global gover 20.10.11
 %global rpmver %{gover}
 %global gitrev b0f5bc36fea9dfb9672e1e9b1278ebab797b9ee0
```
Comment (tjkirch) [inline packages/docker-engine/docker-engine.spec]: I think we need to update this git commit to match the release version - 847da184ad5048b27f5bdf9d53d070f731b43180

File: packages/containerd/containerd.spec [hunk 2]
```diff
@@ -2,7 +2,7 @@
 %global gorepo containerd
 %global goimport %{goproject}/%{gorepo}
 
-%global gover 1.5.5
+%global gover 1.5.8
 %global rpmver %{gover}
 %global gitrev 72cec4be58a9eb6b2910f5d10f1c01ca47d231c0
```
Comment (tjkirch) [inline packages/containerd/containerd.spec]: I think this is 1e5ef943eb76627a6d3b6de8cd1ef6537f393a71 for 1.5.8

**Review by cbgbt — APPROVED** [review]
:boat:

File: packages/docker-engine/docker-engine.spec (hunk 1 above)
Comment (cbgbt) [reply packages/docker-engine/docker-engine.spec]: Nice catch!

**Comment by etungsten** [comment]
Push above addresses @tjkirch 's comments.

**Review by tjkirch — APPROVED** [review]
LGTM, but we should confirm an aarch64 build is OK.

**Review by cbgbt — APPROVED** [review]

**Comment by etungsten** [comment]
Built aws-k8s-1.21 aarch64 image and the node runs pods fine.
Built aws-ecs-1 aarch64 image, and a task runs successfully.

**Review by webern — APPROVED** [review]

**Pull request merged**